### PR TITLE
Removed static keyword from propTypes

### PR DESCRIPTION
**Pull request by avimishra18**
See #588

### CODE REVIEW

File: docs/react/reusable_component.md [hunk 1]
```diff
@@ -152,9 +152,9 @@ import PropTypes from "prop-types";
 And the following static property to the function to validate the properties:
 
 ```jsx
-static propTypes = {
-  question: PropTypes.string.isRequired,
-  answer: PropTypes.string.isRequired
+FaqItem.propTypes = {
+	question: PropTypes.string.isRequired,
```
Comment (stevepiercy) [inline docs/react/reusable_component.md:156]: Please configure your editor to use spaces, never tabs.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
This change is fine with me, but it should be reviewed by a content author.

File: docs/react/reusable_component.md (hunk 1 above)
Comment (avimishra18) [reply docs/react/reusable_component.md]: Please, review now.

**Review by stevepiercy — APPROVED** [review]
LGTM. Ping @jaroel @robgietema @iFlameing for review as authors of the training.

**Review by nileshgulia1 — APPROVED** [review]
LGTM

**Comment by avimishra18** [comment]
If the approval is complete. Should I merge now or someone else will do it?

**Comment by stevepiercy** [comment]
Within the Plone organization, it is customary for the person who submits the PR to merge it after at least one other person as approved it. Besides, you've earned the pleasure of pressing that friendly green button. :grin:

**Pull request merged**